.travis.yml

```diff
@@ -47,3 +47,7 @@ script:
 
 after_success:
   - travis_after_tests_success
+
+branches:
+  except:
+    - /^merge-branch-.*$/
```

Comment:
Is this required for this PR?

Reply:
No is not. I remote it

Reply:
It seems your PR includes an unwanted commit, it is the cause of this addition